docs/Harmonization.md

```diff
@@ -55,21 +55,21 @@ The traditional form of victim notification relates to observations that detail
-|scanner|This type refers to machines which are performing port or vulnerability scanning attempts in general.|These hosts are scanning for vulnerable services to enable threat actors to compromise them. The host doing the scanning are often compromised or infected as well.|
-|spam infrastructure|This type refers to resources which make up a spammer's infrastructure, be it a harvester, dictionary attacker, URL, spam etc.|These hosts will most likely be blocked because they are participating in spamming activities.|
+|defacement|This type of observation refers to hacktivism, which on a technical level is indicative of suspected compromise.|This host is likely to have been compromised by a third party and very often is used for other criminal activities as well.|
+|dropzone|This type of observation refers to a resource which is used to store stolen user data.|Personally identifiable information is often stored unlawfully on these hosts or services.|
+|exploitation|This type of observation refers to hosts attempting to exploit a vulnerable service on a third party system.|These hosts are likely to have been compromised and are trying to break into other hosts or services.|
+|exploit url|This type of observation details an exploit kit, which served through a malicious URL.|These URLs are used by the threat actors as a mechanism to break into vulnerable machines.| 
+|malware configuration|These observations point to resources which update malware infected machines with new configurations.|These hosts or services function as part of threat actor infrastructure and are often compromised by threat actors.|
+|malware infection|This type of observation details hosts which have been observed to call out to a command and control server.|These hosts are likely to have been infected by a piece of malware and are controlled by threat actors.|
+|malware url|An observation referencing a malware URL is the most common resource associated with malware distribution.|These hosts are serving pieces of malware to infect new machines and are often compromised by threat actors.|
+|phishing|This observation type most often refers to a URL or domain name used to defraud the users of their credentials.|These URLs or domain names are served to potential victims to try to steal their credentials to a third party service. These hosts are often also compromised by threat actors.|
+|ransomware|This observation type refers to a specific type of suspected compromise, where the host has been hijacked for ransom by the criminals.|The storage resources of these hosts are encrypted by the criminals for ransom or sabotage. This in turn, may lead to encryption of storage resources for an entire organization.|
```

Comment:
could "storage resources" be e.g. "network shares, databases and other storage" ? Just to be a bit more explicit?

Reply:
Good point. Implemented.